### ISSUE TITLE
[MNG-6372] On Windows Maven can output spurious ANSI escapes such as [0m [0m

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#7461]
**[Stephen Connolly](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=stephenc)** opened **[MNG-6372](https://issues.apache.org/jira/browse/MNG-6372?redirect=false)** and commented

Found during the release vote of Maven 3.5.3 on Windows spurious ANSI partial escape codes such as [0m

A regression introduced by the upgrade of JAnsi to 1.17. Tracked in Jansi as issue #107 https://github.com/fusesource/jansi/issues/107, will be fixed in 1.17.1

notice: another issue from Jansi 1.17 exists, which is failure when run in parallel build with NumberFormatException = see MNG-6382 for details

---

**Affects:** 3.5.3

**Issue Links:**
- [MNG-6382](https://issues.apache.org/jira/browse/MNG-6382) JANSI fails frequently with NumberFormatException when building in parallel

**Remote Links:**
- [Jansi issue #107
](https://github.com/fusesource/jansi/issues/107)

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

https://gitbox.apache.org/repos/asf?p=maven.git&a=commit&h=3c3a5901f35783289d502b67bfc520b8e6936ed8
